### ISSUE TITLE
小建议：在PyPI版本号演进时统一打包各平台（无论代码是否存在变化）

### ISSUE THREAD

**Issue by wyqsmith** [BYVoid/OpenCC#477]
感谢阅读。这并不是个问题，只是一个小建议。

近日刚刚开始使用`OpenCC`，当前遇到的case如下：

个人项目一直使用`pip-tools`管理依赖（锁定依赖库版本号），在`macOS`平台写码，部署在`Linux`的VPS下，使用`Docker`官方的`python`镜像。

在`macOS`平台执行`pip-compile`生成版本号为`opencc==1.1.1`，而此版本无法在`Docker`官方`python`镜像中运行（提示需要`glibc 2.29`或更高版本，官方基于`debian`的镜像中最新只有`glibc 2.28`）。

在当前最新版本`opencc==1.1.1.post1`中，此问题已得到修复（#419）。而因为`PyPI`上的最新版本`1.1.1.post1`只做了`manylinux`的构建，所以`pip-tools`在`macOS`下执行时获取的最新版本仍然是`opencc==1.1.1`。所以无法实现一份`requirements.txt`覆盖全平台。

私以为有两种方案可行：

1. 版本号演进时，无论代码是否存在变化，均发全平台的`wheel`包（大概看了下`.travis.yml`只有`NodeJS`相关构建过程）。
2. 在`pip install`的过程中执行部分编译过程，生成对应环境的`so`文件，与环境一定程度上解耦（仅约束最低编译环境）。

若需要相关代码`Pull Request`请回复此`PR`，会在业余时间尽绵薄之力。

以上。万分感谢。祝好。

**Comment by sgalal** [comment]
@mingruimingrui

**Comment by mingruimingrui** [comment]
I agree. We should also probably set up the codebase to deploy using CI/CD that way we can ensure releases are done properly.
I'm not exactly in a position to greenlight a PR.
@sgalal @BYVoid

**Comment by sgalal** [comment]
> We should also probably set up the codebase to deploy using CI/CD that way we can ensure releases are done properly.

Sure. That would be better.

**Comment by BYVoid** [comment]
Agreed. Please contribute to https://github.com/BYVoid/OpenCC/blob/master/BUILD.md

**Comment by frankslin** [comment]
https://pypi.org/project/OpenCC/1.3.1/ 應已解決。
